reactioncommerce/reaction
# 1987 Panel Expander Arrow Overlapping on Dropdown

Resolves [1987](https://github.com/reactioncommerce/reaction/issues/1987)

**Testing**
- Clone this Branch and run `reaction` on your local machine
- Sign In with admin credentials
- Open Dashboard and select `Localization and i18n`
- Select the `Base Language` drop-down
- Observe that the `Languages` panel expander does not overlap on the dropdown

**Screenshot**
<img width="376" alt="screen shot 2017-05-04 at 4 02 47 pm" src="https://cloud.githubusercontent.com/assets/10584060/25710120/3050fc30-30e3-11e7-8bb9-f42579b74f2e.png">
